### PR TITLE
Update claim fee logic on boost.claimed

### PR DESCRIPTION
**Pull request by pntemi**
2 updates:
1. Boost recently removed the claim fee from our protocol. This PR is to update the logic to reflect the current fee.
2. We had a bug in our logging where all Drakula's boost with action on Base are logged as Optimism instead.

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @pntemi

File: models/boost/boost_claimed.sql [hunk 1]
```diff
@@ -44,7 +44,10 @@ quest_claims_enriched as (
         coalesce(b.reward_token_address, c.reward_token_address) reward_token_address,
         claim_tx_hash,
         c.block_time,
-        coalesce(claim_fee_eth, 0.000075) claim_fee_eth,
+        case 
+            when block_time >= timestamp '2024-06-06 16:00:00' then 0.0
```
Comment (Hosuke) [inline models/boost/boost_claimed.sql:48]: We may introduce these timestamps as jinja variables.

Comment (jeff-dude) [reply models/boost/boost_claimed.sql:48]: @Hosuke do you mind pushing a commit to show how it's done? then @pntemi can leverage these things moving forward 🤝

Comment (Hosuke) [reply models/boost/boost_claimed.sql]: Just added.

Comment (jeff-dude) [reply models/boost/boost_claimed.sql]: i think the variables need single quotes around them, went ahead and pushed. let's see CI run now.

Comment (jeff-dude) [reply models/boost/boost_claimed.sql]: looks to work now

**Review by jeff-dude — APPROVED** [review]

File: models/boost/boost_claimed.sql (hunk 1 above)
Comment (Hosuke) [reply models/boost/boost_claimed.sql]: Forgot to push the local commits.

**Pull request merged**